nsv13/code/modules/clothing/clothing.dm

```diff
@@ -0,0 +1,41 @@
+//parent append extra vars here
+/obj/item/clothing
+	var/list/species_restricted = null //Added variable to it, basically a restriction list.
```

Comment:
Would it not be better to use defines rather than strings? I'm fine with this either way

Reply:
It draws from the species names that already exist, I'd have to make a define for everything that exists wouldn't I?